src/app/components/PageLayout/NetworkHeader.tsx

```diff
@@ -20,24 +21,6 @@ const StyledNetworkHeader = styled(Box)(({ theme }) => ({
   padding: 0,
   border: 'none',
   borderRadius: 'none',
```

Comment:
Looking at https://github.com/oasisprotocol/explorer/pull/256, I think `position: 'relative'` above can be reverted too

Reply:
thx, it looks like I can remove `StyledNetworkHeader` completly.